build.zig

```diff
@@ -43,6 +43,16 @@ pub fn build(b: *std.build.Builder) void {
         const test_step = b.step("test", "Run the unit tests");
         test_step.dependOn(&unit_tests.step);
     }
+
+    {
+        const benchmark = b.addExecutable("eytzinger_benchmark", "src/eytzinger_benchmark.zig");
+        benchmark.setTarget(target);
+        benchmark.setBuildMode(mode);
```

Comment:
Do we need to set this to `ReleaseSafe` explicitly to make sure the benchmark is not run in `Debug` mode?